src/core/settings.h

```diff
@@ -382,20 +382,97 @@ enum class GPUAccuracy : u32 {
     Extreme = 2,
 };
 
+extern bool configuring_global;
+
+template <typename Type>
+class Setting final {
+public:
+    Setting() {
```

Comment:
```suggestion
    Setting() = default;
```

After previous suggestions are applied.